nixos/modules/services/web-apps/ifm.nix

```diff
@@ -0,0 +1,81 @@
+{ config, lib, pkgs, ...}:
+let
+  cfg = config.services.ifm;
+
+  version = "4.0.2";
+  src = pkgs.fetchurl {
+    url = "https://github.com/misterunknown/ifm/releases/download/v${version}/cdn.ifm.php";
+    hash = "sha256-37WbRM6D7JGmd//06zMhxMGIh8ioY8vRUmxX4OHgqBE=";
+  };
```

Comment:
This should be a package

Reply:
that will enable r-ryantm bumps, and setting `meta.license` and `meta.sourceProvenance`

Reply:
I remarked on this issue above (https://github.com/NixOS/nixpkgs/pull/328205#pullrequestreview-2186639824) but concluded it would be fine for such a small package (which is basically just one `php` file). 
However I haven't considered the `meta` fields and automatic package updates by ryan. @litchipi maybe you could do a follow-up PR which separates the package `ifm` from the module `ifm`?

Reply:
Working on it, sorry didn't thought of these problems.

However, the name `ifm` is already taken as a package name (in `pkgs/tools/graphics/ifm`), so I don't really know what to name it...